qiskit_experiments/library/characterization/qubit_spectroscopy.py

```diff
@@ -88,7 +87,6 @@ def _default_analysis_options(cls) -> Options:
         options.normalization = True
         options.xlabel = "Frequency"
         options.ylabel = "Signal (arb. units)"
-        options.xval_unit = "Hz"
```

Comment:
this `xval_unit` is necessary because this represents an axis label of output figure.